hugo.yaml

```
@@ -16,6 +16,11 @@ languages:
     languageCode: en-US
     params:
       description: The OpenTelemetry Project Site
+  es:
+    languageName: Español
+    languageCode: es-ES
```

```suggestion
    languageCode: es
```
just use the generic code for spanish language and don't use the ES for region Spain

---

If you are not specifying a region, then drop the line altogether.